### DISCUSSION TITLE
What is the best free resources to learn React

### DISCUSSION THREAD

**Discussion by 0xsimpi** [ows-web/hacktoberfest22#46] [Q&A] [answered]
I have just started learning javascript and react and i am very passionated about react. please provide some resources from where i can learn my self rather then buying courses on online plateform.

**Comment by 0xsonu** [comment, marked as answer]
> I have just started learning javascript and react and i am very passionated about react. please provide some resources from where i can learn my self rather then buying courses on online plateform.

Hey @simpikri , These are following resources you can follow. these are free.

- [x]  [FreeCodeCamp React List](https://www.freecodecamp.org/news/search/?query=react)
- [x]  [React Official Documentation](https://reactjs.org/tutorial/tutorial.html)
- [x]  [JavaScript30](https://javascript30.com/)
- [x]  [Javascript for Begineers](https://www.freecodecamp.org/news/full-javascript-course-for-beginners/) 
- [x]  [Javascript Official MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript)


in future i will update list.

**Reply by 0xsimpi** [reply]
@SonuKumar81800 Thank you sir for these resources.
